test/controllers/login_controller_test.rb

```diff
@@ -45,7 +44,7 @@ class LoginControllerTest < ActionController::TestCase
         before_count = User.count
         post :urs, provider: :urs
         after_count = User.count
-        assert(after_count.must_equal(before_count + 1));
+        # assert(after_count.must_equal(before_count + 1));
```

Comment:
does this mean nothing is actually being tested here?

Reply:
speaking with Chris this is commented out to be addressed later.

I think it might be nice to add a comment/TODO to that effect.

Reply:
it is commited and pushed now thanks